src/sentry/models/commitauthor.py

```diff
@@ -20,6 +20,13 @@ class Meta:
 
     __repr__ = sane_repr("organization_id", "email", "name")
 
+    @classmethod
```

Comment:
Would it make sense to just put this on the manager? Then you don't need to change any other code.